### DISCUSSION TITLE
Not seeing my own created score in the Ipad application

### DISCUSSION THREAD

**Discussion by abregar** [musescore/MuseScore#22259] [Q&A] [answered]
Hi,
being PRO Musescore user for some time, I have recently created my first score with the windows based machine (Musescrore version 4.2.1), saved it to the cloud and then expected to be visible in my library in the Ipad application. Which did not happen.
Reading through forums there are already some mentions from people having same experience, but couldn't find out through manuals and now also opened issues here at Github, whether this is a bug or just some hidden configuration issue?
kr

**Comment by MarcSabatella** [comment, marked as answer]
MuseScore 4 is not yet supported in the mobile apps.  That's been announced in the beta that should become available this week if you signed up for it, otherwise I assume it will be pretty soon in general release.

**Reply by abregar** [reply]
Tnx for the information. May I ask where the beta was announced, as I cant see anything in the forum https://musescore.org/en/forum ?

**Reply by Jojo-Schmitz** [reply]
It's been announced where it belongs: https://musescore.com/groups/musescore-on-ios/discuss/5248018
